tests/tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+function dashboard_admin_login {
+    touch $COOKIES
+    $CURL_BIN $LOGIN_URL > /dev/null
+    DJANGO_TOKEN="csrfmiddlewaretoken=$(grep csrftoken $COOKIES | \
+    sed 's/^.*csrftoken[[:blank:]]*//')"
+    $CURL_BIN -d "$DJANGO_TOKEN;username=$USERNAME;password=$PASSWORD" \
+    -X POST $LOGIN_URL
+}
+
+function run_dashboard_tests {
+    # This tests the login was successful being able to 
+    # access this page also means redis is running.
+    $CURL_BIN --head http://dashboard.openwisp.org/admin/ | grep -q "200 OK" && \
+    echo "SUCCESS: Admin login successful!" || \
+    (echo "ERROR: Admin login failed!"; exit 1)
+}
+
+function init_dashoard_tests {
+    LOGIN_URL=http://dashboard.openwisp.org/admin/login/
+    USERNAME='admin'
+    PASSWORD='admin'
+    COOKIES=cookies.txt
+    CURL_BIN="curl -s -c $COOKIES -b $COOKIES -e $LOGIN_URL"
+    # Check if openwisp-dashboard is reachable
+    curl -s --head $LOGIN_URL  | grep -q "200 OK" && \
+    echo "SUCCESS: openwisp-dashboard login page reachable!" || \
+    (echo "ERROR: openwisp-dashboard login page not reachable!"; exit 1)
+    dashboard_admin_login # Login to django-admin
+    run_dashboard_tests # Run CI tests
+    rm $COOKIES # Delete cookies file created during testing
+}
```

Comment:
if you can write a test that creates a new user in the system, that would be SUPER, because we had many issues with that recently

Reply:
Not sure if its what your looking for but I use the django-freeradius batch_add_user in my travis tests.
>docker-compose run --rm -v $PWD/scripts/users.csv:/users.csv django python manage.py batch_add_users --name users --file /users.csv

Reply:
I meant to add an user from the web interface

Reply:
That's in the kanban board, I will implement that after the postfix service container is ready and running! :-)

Reply:
great :+1: thanks for clarifying